DFC.App.Triagetool/Properties/launchSettings.json

```diff
@@ -17,11 +9,18 @@
     },
     "DFC.App.Triagetool": {
       "commandName": "Project",
-      "launchBrowser": true,
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
-      "applicationUrl": "https://localhost:5001;http://localhost:5000"
+      "applicationUrl": "https://localhost:6001;http://localhost:6000"

```

Comment:
@hnandra-DFE do we know the impact of this on other services?

Reply:
This has no impact on other services - only changes the port when running locally. Probably due to ports clashing when debugging multiple applications at the same time. I don't believe this code change is required though.